nuget/Microsoft.Windows.CsWinRT.nuspec

```diff
@@ -23,13 +23,15 @@
     <file src="Microsoft.Windows.CsWinRT.props" target="build"/>
     <file src="Microsoft.Windows.CsWinRT.targets" target="build"/>
     <file src="Microsoft.Windows.CsWinRT.Prerelease*.targets" target="build"/>
+    <file src="Microsoft.Windows.CsWinRT.Authoring.targets" target="build"/>
+    <file src="Microsoft.Windows.CsWinRT.Authoring.targets" target="buildTransitive"/>
     <file src="readme.txt"/>
     <file src="$net5_runtime$" target="lib\net5.0\"/>
     <file src="$source_generator$" target="analyzers\dotnet\cs\"/>
-    <file src="$winrt_host_x64$" target ="native\x64"/>
-    <file src="$winrt_host_x86$" target ="native\x86"/>
-    <file src="$winrt_host_arm$" target ="native\arm"/>
-    <file src="$winrt_host_arm64$" target ="native\arm64"/>
+    <file src="$winrt_host_x64$" target ="runtimes\win-x64\native"/>
+    <file src="$winrt_host_x86$" target ="runtimes\win-x86\native"/>
+    <file src="$winrt_host_arm$" target ="runtimes\arm\native"/>
```

Comment:
any reason why we dropped the win- convention for arm.  https://docs.microsoft.com/en-us/dotnet/core/rid-catalog#windows-rids